reverse_order_range.py

```diff
@@ -0,0 +1,5 @@
+list1 = []
+for i in range(1,40):
+	list1.append(i)
+list1.reverse()
+print list1
```

Comment:
Hi Saru,
The goal is to get the reverse range of numbers for iterations. Not for printing numbers in reverse order.